CombatTagPlus/src/main/java/net/minelink/ctplus/task/TagUpdateTask.java

```diff
@@ -51,6 +51,9 @@ public void run() {
             if (!plugin.getSettings().getUntagMessage().isEmpty()) {
                 player.sendMessage(plugin.getSettings().getUntagMessage());
             }
+
+            plugin.getTagManager().untag(player.getUniqueId());
```

Comment:
Instead of making this change here, wouldn't it make more sense to fire the untag event in the getTag method, as thats when the tag _actually_ expires?